internal/engine/wazevo/testcases/testcases.go

```diff
@@ -1512,6 +1512,35 @@ var (
 			}}},
 		},
 	}
+
+	VecArithmetics = TestCase{
+		Name: "vec_arithmentics",
+		Module: SingleFunctionModule(wasm.FunctionType{
+			Params:  []wasm.ValueType{v128, v128},
+			Results: []wasm.ValueType{v128, v128, v128, v128},
+		}, []byte{
+			wasm.OpcodeLocalGet, 0,
+			wasm.OpcodeLocalGet, 1,
+			wasm.OpcodeVecPrefix, wasm.OpcodeVecI64x2Add,
+			// wasm.OpcodeDrop,
```

Comment:
if I add `drop`, then the ssa instructions are retained (until the backend, where obviously are all pruned as it's dead code), otherwise it returns unreachable instructions 🤔 probably I am missing something silly 😬  

        blk0: (exec_ctx:i64, module_ctx:i64, v2:v128, v3:v128)
        	v4:v128 = VIadd.i64x2 v2, v3
        	Exit exec_ctx, unreachable

Reply:
i pushed the fix for you (a bug in PC advancement) https://github.com/tetratelabs/wazero/pull/1698/commits/87cf00ed3120e2c6c89833b8a86808459e9d846b 


btw curious did you really pass spectests with this bug? I guess no and otherwise how sloppy they are 😄